# Multimodal Single Cell Data Integration

## Publications

**[A]: Algorithm; [D]: Datasets; [R]: Review**

### 2021
* [R] Argelaguet, R., Cuomo, A. S. E., Stegle, O. & Marioni, J. C. Computational principles and challenges in single-cell data integration. Nat Biotechnol 1–14 (2021)
* [A] Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573-3587.e29 (2021). 
   * The Weighted nearest neighbor (WNN) method used in Seurat V4
* [A] Gao, C. et al. Iterative single-cell multi-omic integration using online learning. Nat Biotechnol 1–8 (2021)
* [A] Singh, R., Hie, B. L., Narayan, A. & Berger, B. Schema: metric learning enables interpretable synthesis of heterogeneous single-cell modalities. Genome Biol 22, 131 (2021).
* [A] Cantini, L. et al. Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer. Nat Commun 12, 124 (2021).
* [A] Lotfollahi, M. et al. Mapping single-cell data to reference atlases by transfer learning. Nat Biotechnol 1–10 (2021) doi:10.1038/s41587-021-01001-7.
   * The scArc paper. The first transfer learning application on scRNA-seq integration
* [A] Kuchroo, M., Godavarthi, A., Wolf, G. & Krishnaswamy, S. Multimodal data visualization, denoising and clustering with integrated diffusion. Arxiv (2021).
* [A] Martinez-de-Morentin, X., Khan, S. A., Lehmann, R., Tegner, J. & Gomez-Cabrero, D. Machine Translation between paired Single Cell Multi Omics Data. Biorxiv 2021.01.27.428400 (2021) doi:10.1101/2021.01.27.428400.
* [D] Swanson, E. et al. Simultaneous trimodal single-cell measurement of transcripts, epitopes, and chromatin accessibility using TEA-seq. Elife 10, e63632 (2021).
   * The TEA-seq (Transcription, Epitopes, and Accessibility) paper
* [A] Minoura, K., Abe, K., Nam, H., Nishikawa, H. & Shimamura, T. A mixture-of-experts deep generative model for integrated analysis of single-cell multiomics data. Cell Reports Methods 1, 100071 (2021).
  
### 2020
* [A] Cao, K., Bai, X., Hong, Y. & Wan, L. Unsupervised topological alignment for single-cell multi-omics integration. Bioinformatics 36, i48–i56 (2020)  
* [A] Uzun, Y., Wu, H. & Tan, K. Predictive modeling of single-cell DNA methylome data enhances integration with transcriptome data. Genome Res 31, gr.267047.120 (2020).
* [A] Jin, S., Zhang, L. & Nie, Q. scAI: an unsupervised approach for the integrative analysis of parallel single-cell transcriptomic and epigenomic profiles. Genome Biol 21, 25 (2020). 
* [A] Argelaguet, R. et al. MOFA+: a statistical framework for comprehensive integration of multi-modal single-cell data. Genome Biol 21, 111 (2020).
* [A] Stark, S. G. et al. SCIM: universal single-cell matching with unpaired feature sets. Bioinformatics 36, i919–i927 (2020).
* [A] Cao, K., Bai, X., Hong, Y. & Wan, L. Unsupervised topological alignment for single-cell multi-omics integration. Bioinformatics 36, i48–i56 (2020).
* [A] Ma, S. et al. Chromatin Potential Identified by Shared Single-Cell Profiling of RNA and Chromatin. Cell 183, 1103-1116.e20 (2020).
* [D] Cuomo, A. S. E. et al. Single-cell RNA-sequencing of differentiating iPS cells reveals dynamic genetic effects on gene expression. Nat Commun 11, 810 (2020).
* [A] Zuo, C. & Chen, L. Deep-joint-learning analysis model of single cell transcriptome and open chromatin accessibility data. Brief Bioinform 22, bbaa287- (2020).
  
### 2019
* [D] Argelaguet, R. et al. Multi-omics profiling of mouse gastrulation at single-cell resolution. Nature 576, 487–491 (2019).
* [D] Luo, C. et al. Single nucleus multi-omics links human cortical cell regulatory genome diversity to disease risk variants. Biorxiv 2019.12.11.873398 (2019) doi:10.1101/2019.12.11.873398.
* [D] Zhu, C. et al. Joint profiling of histone modifications and transcriptome in single cells from mouse brain. Nat Methods 18, 283–292 (2021).
* [A] Liu, J., Huang, Y., Singh, R., Vert, J.-P. & Noble, W. S. Jointly embedding multiple single-cell omics measurements. Biorxiv 644310 (2019) doi:10.1101/644310.
* [D] Chen, S., Lake, B. B. & Zhang, K. High-throughput sequencing of the transcriptome and chromatin accessibility in the same cell. Nat Biotechnol 37, 1452–1457 (2019).    
* [D] Liu, L. et al. Deconvolution of single-cell multi-omics layers reveals regulatory heterogeneity. Nat Commun 10, 470 (2019). 
   * The scCAT-seq (single-cell chromatin accessibility and transcriptome sequencing) paper
* [D] Zhu, C. et al. An ultra high-throughput method for single-cell joint analysis of open chromatin and transcriptome. Nat Struct Mol Biol 26, 1063–1070 (2019).
* [A] Shi, Y., Siddharth, N., Paige, B. & Torr, P. H. S. Variational Mixture-of-Experts Autoencoders for Multi-Modal Deep Generative Models. Arxiv (2019).
  
    
    
### 2018
* [D] Duren, Z. et al. Integrative analysis of single-cell genomics data by coupled nonnegative matrix factorizations. Proc Natl Acad Sci U S A 115, 7723–7728 (2018).
* [D] Clark, S. J. et al. scNMT-seq enables joint profiling of chromatin accessibility DNA methylation and transcription in single cells. Nat Commun 9, 781 (2018).
* [D] Cao, J. et al. Joint profiling of chromatin accessibility and gene expression in thousands of single cells. Science 361, 1380–1385 (2018).
   * The sci-CAR paper
    
  
### 2017
* [D] Stoeckius, M. et al. Simultaneous epitope and transcriptome measurement in single cells. Nat Methods 14, 865–868 (2017). 
  * The CITE-seq paper. 
* [A] Welch, J. D., Hartemink, A. J. & Prins, J. F. MATCHER: manifold alignment reveals correspondence between single cell transcriptome and epigenome dynamics. Genome Biol 18, 138 (2017).
  

### 2016 
* [D] Angermueller, C. et al. Parallel single-cell sequencing links transcriptional and epigenetic heterogeneity. Nat Methods 13, 229–232 (2016). 
   * The scM&T-seq paper.
* [D] Cheow, L. F. et al. Single-cell multimodal profiling reveals cellular epigenetic heterogeneity. Nat Methods 13, 833–836 (2016).


  
  
  
  
  
